### DISCUSSION TITLE
React box `Can't resolve '../../contracts/SimpleStorage.json'`

### DISCUSSION THREAD

**Discussion by Prashant-Mishra-12569** [ConsenSys-archive/truffle#5280] [General Q&A]
when I run this command npm run start warning showing, showing this error:
```
Compiled with warnings.

./src/contexts/EthContext/EthProvider.jsx
Module not found: Can't resolve '../../contracts/SimpleStorage.json' in 'C:\Users\glc\Desktop\truffle\frontend\client\src\contexts\EthContext'

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before
```

**Comment by cliffoo** [comment]
Hey @Prashant-Mishra-12569, thanks for opening this discussion.

This is by design: If you go to where the react app is running (most likely localhost:3000), there will be preparation steps that walk you through setup with MetaMask, Ganache, and deploying the `SimpleStorage` contract with Truffle. The deployment creates the missing `SimpleStorage.json` build artifact, which removes the compiler warning and lets you interact with the contract under `See it in action` section.

**Reply by Prashant-Mishra-12569** [reply]
I can't understand

**Reply by cliffoo** [reply]
1. Follow [`installation steps`](https://github.com/truffle-box/react-box#installation), you will have a react app running (most likely at localhost:3000).
2. Follow steps under `Preparation` on the react app.

**Reply by Prashant-Mishra-12569** [reply]
where I find SimpleStorage.json file please provide me
please please

**Comment by Prashant-Mishra-12569** [comment]
please give me simpleStorage.json file please
